openghg/retrieve/_search.py

```diff
@@ -102,7 +156,9 @@ def search_surface(
         start_date=start_date,
         end_date=end_date,
         data_source=data_source,
-        **kwargs,
+        network=network,
+        sampling_height=sampling_height,
+        # **kwargs,
     )
 
     # TODO - remove this cast once we've updated search to ensure return of SearchResults object
```

Comment:
Just noticed this comment - can the `cast()` be removed now?

Reply:
Done.